Sources/Build/BuildPlan.swift

```diff
@@ -578,16 +578,13 @@ public final class SwiftTargetBuildDescription {
         // Do nothing if we're not generating a bundle.
         guard let bundlePath = self.bundlePath else { return }
 
-        // Compute the basename of the bundle.
-        let bundleBasename = bundlePath.basename
-
         let stream = BufferedOutputByteStream()
         stream <<< """
         import class Foundation.Bundle
 
         extension Foundation.Bundle {
             static var module: Bundle = {
-                let bundlePath = Bundle.main.bundlePath + "/" + "\(bundleBasename)"
+                let bundlePath = "\(bundlePath)"
```

Comment:
Wouldn't this hardcode the absolute path into each artifact (also non-tests)?

Reply:
I'm not familiar with how `bundlePath` is derived so that is possible. I wasn't able to confirm this on an iOS device itself as Xcode won't let me install it on my test device with a custom toolchain. Is it possible to check if it's compiling a test target?

Reply:
Be aware that the Xcode integration of packages does not use SwiftPM from the toolchain, you will only see an effect of your changes in commandline SwiftPM.

Reply:
Right, but in this case, wouldn't this also change the path that's built into a package target that's linked into an executable, meaning that if the executable and its associated resource bundles are later moved, the bundles can't be found?

Reply:
> Right, but in this case, even when just building with `swift build`, wouldn't this also change the path that's built into a package target that's linked into an executable, meaning that if the executable and its associated resource bundles are later moved, the bundles can't be found?

That seems likely, when I tested I did move the executable but not the resource bundle. Would it make sense to conditionally check if the current Bundle.main path is located in an Xcode path and fallback to the hard coded path? 

@neonichu I'm guessing that Xcode has its own method for computing Bundle.module? I had gone down a rabbit hole of trying to run my tests with xcodebuild on my package, but had other issues altogether there.